### PR TITLE
Voter Dapp Header

### PR DESCRIPTION
**Pull request by rosalindstengle**
As we discussed, the styling on this component is still funky, but structurally it works just fine. Let me know what you think of the wording particularly on the token balance bit.

### CODE REVIEW

File: voter-dapp/src/App.js [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
+import Header from "./Header.js";
 import ActiveRequests from "./ActiveRequests.js";
-import ResolvedRequests from "./ResolvedRequests.js";
+import { red, grey } from "@material-ui/core/colors";
+import AppBar from "@material-ui/core/AppBar";
 
 function App() {
+  const theme = createMuiTheme({
+    palette: {
+      primary: red, //TODO --> find actual UMA red
```
Comment (mrice32) [inline voter-dapp/src/App.js]: UMA red is `#ff4a4a`.

See the press kit for all UMA colors and styles: https://github.com/UMAprotocol/website/blob/master/press-kit/UMA_Style_Guide_v2.pdf

You should be able to plug in any hex color by just setting the property to it:
```
primary: "#ff4a4a"
```

File: voter-dapp/src/App.js [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+      secondary: grey
```
Comment (mrice32) [inline voter-dapp/src/App.js]: nit: there should be a list of secondary colors in the press kit too. See the [sponsor dapp styling](https://github.com/UMAprotocol/protocol/blob/a4b6c69bc521e554493af4f330e04af1def20f21/sponsor-dapp/src/App.js#L32-L57) for how we did this there. I only show that example to illustrate how to do this - don't feel obligated to match the look and feel of the sponsor dapp.

File: voter-dapp/src/Header.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          <img src={logo} alt="Logo" style={{ padding: 15 }} height="30" />
+        </Grid>
+        <Grid item>
+          <h1>Vote Requests</h1>
+        </Grid>
+        <Grid item>
+          <ul id="nav-mobile" class="right hide-on-med-and-down" style={{ float: "right" }}>
+            <li>User Addr = {account}</li>
+            <li>
+              {balance} tokens of {supply} supply is {!(balance / supply) ? 0 : (balance / supply) * 100}%{" "}
```
Comment (mrice32) [inline voter-dapp/src/Header.js]: What happens when balance and/or supply haven't been returned by drizzle yet? In that case both will be undefined, right? How does that look when rendering? Should we replace them with something reasonable in that case?

Comment (rosalindstengle) [reply voter-dapp/src/Header.js]: what do we usually do before drizzle has loaded? displaying nothing is probably more confusing to the user

**Pull request merged**